recipes/omsl/conda_build_config.yaml

```diff
@@ -0,0 +1,13 @@
+mslversion:
+  - 3.2.3
+  - 4.0.0
```

Comment:
Do not use the conda_build_config to build multiple versions of osml. Start with the older version, then once this PR is merged, open a PR on the feedstock to update to the newer version. If you need to maintain v3.x and v4.x simultaneously, then create a branch for each in the feedstock. Every commit on the feedstock is published.

Reply:
OK, understood, thanks for explaining this. Indeed, I will have to maintain v3.x and v4.x because they are not compatible and v3.x is still widely used in production. So I removed conda_build_config.